Makefile.am

```diff
@@ -146,6 +146,17 @@ shellcheck:
 		echo "skipping shellcheck because shellcheck is not installed"; \
 	fi
 
+PHONY += checkbashisms
+checkbashisms:
+	@if type checkbashisms > /dev/null 2>&1; then \
+		checkbashisms -n -p -x \
+			$$(find ${top_srcdir} -name '.git' -prune -o -name 'build' -prune -o \
+			-type f ! -name 'config*' \
+			-exec bash -c 'awk "NR==1 && /\#\!.*bin\/sh.*/ {print FILENAME;}" "{}"' \;); \
```

Comment:
We're going to want to additionally exclude the following autotools provided scripts which we haven't modified.  These are the only pure scripts in the `config` directory so it's probably cleanest to exclude the whole directory.  

```
config/install-sh
config/depcomp
config/missing
config/config.sub
config/config.guess
config/ltmain.sh
config/compile
```

Reply:
:+1: I did this in the other PR but I hadn't updated this one yet.

Reply:
Fixed.